.gitignore

```diff
@@ -136,3 +136,7 @@ dmypy.json
 *.jou
 *.ods
 *.vtk
+*.cub
+*.cub5
+*.h5
+logMem.sh
```

Comment:
where does this file come from?

Reply:
the cubit files and h5 are from testing different things, I suppose I should just delete these items out of my parastell directory rather than cluttering the gitignore?

logMem.sh definitely shouldn't be in here

Reply:
I was specifically asking about the `logMem.sh` file.  It seemed a little specific.